### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by yihezkel**
I only made the fixes that I cold do quickest; there's much to do to improve this article.

## Category

- [ ] Content fix
- [ ] New article
- [x] Example checked item (*delete this line*)

> **DELETE THIS LINE BEFORE SUBMITTING** | *For the above list, an empty checkbox is [ ] as in <kbd>[</kbd><kbd>SPACE</kbd><kbd>]</kbd>. A checked checkbox is [x] with no space between the brackets. Use the `PREVIEW` tab at the top right to preview the rendering before submitting your issue.*

## Related issues

- fixes #issuenumber
- partially #issuenumber
- mentioned in #issuenumber

> **DELETE THIS LINE BEFORE SUBMITTING** | *If this fixes (aka: closes) or references an issue, please reference it here. This helps maintaining the issue list as it will (1) link the PR to the issue & (2) automatically close the issue when this PR is merged in.*

## What's in this Pull Request?

> **DELETE THIS LINE BEFORE SUBMITTING** | *Please describe the changes in this PR. Sample description or details around bugs which are being fixed.*

## Submission guidelines

> - **!!IMPORTANT!!** - All submissions must complete the baseline sections included in this template. Ignoring or deleting this template may result in closing the issue with the label **type:incomplete-submission**.
> - Follow our guidance on [How To Create Good Pull Requests](https://github.com/SharePoint/sp-dev-docs/wiki/How-to-Create-Good-Pull-Requests).
> - Target the `main` branch of this repo.
> - When changing a page, ensure you update the `ms.date` front matter wih the current date in the format `MM/DD/YYYY`.
> - Review all build checks and address the automated errors, warnigns, and suggestions.
> - *NOTE: The live site is based on the `live` branch. Site owners periodically refresh `live` branch from the `main` branch so merged PRs won't immediately appear on the live site. Please be patient to see your changes appear on the live site.*
>
> **DELETE THIS SECTION BEFORE SUBMITTING**

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[1d6f171](https://github.com/yihezkel/sp-dev-docs/commits/1d6f1712ec2601b65939bc42fc4ee2809890db95)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/solution-guidance/security-apponly-azuread.md](https://github.com/yihezkel/sp-dev-docs/blob/patch-1/docs/solution-guidance/security-apponly-azuread.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azuread?branch=pr-en-us-8430) |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202209110933406447-8430/BuildReport?accessString=e42992d28b2a7d8c9f923978319a511e7707694e05c98fb06760822d58ecd4a5).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/solution-guidance/security-apponly-azuread.md [hunk 1]
```diff
@@ -10,21 +10,21 @@ ms.localizationpriority: medium
...
 
 > [!IMPORTANT]
 > Azure Access Control (ACS), a service of Azure Active Directory (Azure AD), has been retired on November 7, 2018. This retirement does not impact the SharePoint Add-in model, which uses the `https://accounts.accesscontrol.windows.net` hostname (which is not impacted by this retirement). For more information, see [Impact of Azure Access Control retirement for SharePoint Add-ins](https://developer.microsoft.com/office/blogs/impact-of-azure-access-control-deprecation-for-sharepoint-add-ins).
 
 ## Setting up an Azure AD app for app-only access
 
-In Azure AD when doing app-only you typically use a certificate to request access: anyone having the certificate and its private key can use the app and the permissions granted to the app. Below steps walk you through the setup of this model.
+In Azure AD when accessing app-only, you typically use a certificate to request access: anyone having the certificate and its private key can use the app and the permissions granted to the app. Below steps walk you through the setup of this model.
```
Comment (andrewconnell) [inline docs/solution-guidance/security-apponly-azuread.md:20]: Incorrect grammar... you don't "access" app-only, you implement or use.

**Review by andrewconnell — CHANGES_REQUESTED** [review]
When you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.

**Comment by andrewconnell** [comment]
Also, in the future, please follow the instructions in the PR template when submitting a new PR. In the future, ignoring the instructions/guidance in the PR template will result in rejecting the PR without review. It helps us review submissions in a timely manner.

**Comment by andrewconnell** [comment]
Closed to no activity/response by author

File: docs/solution-guidance/security-apponly-azuread.md (hunk 1 above)
Comment (yihezkel) [reply docs/solution-guidance/security-apponly-azuread.md:20]: @andrewconnell  I was referring to the AAD call. Wouldn't that be an access request?

**Comment by yihezkel** [comment]
@andrewconnell 
"Also, in the future, please follow the instructions in the PR template when submitting a new PR. In the future, ignoring the instructions/guidance in the PR template will result in rejecting the PR without review. It helps us review submissions in a timely manner."

I think this is a bit harsh. I didn't **ignore** the instructions. I tried to follow the steps, but apparently I missed one (the date). Is it really better to reject a PR with good suggestions to improve our docs because of a step missing?

**Comment by yihezkel** [comment]
@andrewconnell "Closed to no activity/response by author"
I apologize, just got back this week after 3 weeks off.
Do you not want any of these docs improvements?

**Comment by andrewconnell** [comment]
@yihezkel said:
> I think this is a bit harsh. I didn't **ignore** the instructions. I tried to follow the steps, but apparently I missed one (the date). Is it really better to reject a PR with good suggestions to improve our docs because of a step missing?

Sorry if it came across as harsh... wasn't the intent. The PR template has sections to fill in the blanks as you can see above... things to indicate what the PR is about, what changes it includes if it's a content update or a new doc, etc. None of that stuff was addressed and seemed ignored, meaning it takes longer for reviewers to understand what this is about.

That comment was simply included as a request as this seemed to be your first PR submission.

But with no response for nearly 4 weeks after a review & a request for changes, the issue was closed as it seemed we'd not get any follow-up to the requested changes.

> Do you not want any of these docs improvements?

We do, but as you said, there are a lot of changes to apply to the doc as it's outdated. We'd rather have a complete update than a partial one because a partial one implies it's been updated and is now current. That confuses consumers of the doc who think "oh, if this was updated in October, then everything is current".

**Pull request closed without merging**